src/views/PanelViews/DepartmentViews/DepartmentCreationView/DepartmentCreationView.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { DepartmentFormView } from '../components';
+
+export function DepartmentCreationView() {
+    const handleCreation = () => {};
```

Comment:
I don't really see the reason to have the placeholder functions like this. Just don't pass anything, if it is not ready